scripts/run-confluece-container.sh

```diff
@@ -1,20 +1,19 @@
-#!/bin/bash
+#!/usr/bin/env bash
 #-e  Exit immediately if a command exits with a non-zero status.
 set -e
 
 function usage
 {
     local scriptName=$(basename "$0")
-    echo "usage: ${scriptName} ENV=VALUE ENV2=VALUE"
+    echo "usage: ${scriptName} x.y.z ENV=VALUE ENV2=VALUE"
```

Comment:
No need for the user to know that the main environment variable is called CONFLUENCE_VERSION